src/toil/lib/docker.py

```diff
@@ -0,0 +1,255 @@
+import base64
+import logging
+import subprocess
+
+import os
+from bd2k.util.exceptions import require
+
+_logger = logging.getLogger(__name__)
+
+
+def dockerCall(job,
+               tool,
+               parameters=None,
+               workDir=None,
+               dockerParameters=None,
+               outfile=None,
+               defer=None):
+    """
+    Wrapper call to Docker. See docstring for _docker()
```

Comment:
I had a comment lower down about making the `_docker` documentation into module level docs. LMK if that makes sense.